model-optimizer/extensions/middle/L2NormToNorm.py

```diff
@@ -82,6 +82,17 @@ def replace_pattern(self, graph: Graph, match: dict):
             log.debug('The value of the "maximum_y_data" is not defined or is not constant')
             return
 
+        # We need to check axes which performed reduction because IE supports reduction only along HW and HWC axes.
```

Comment:
what about 5D rank tensors. May it be better to say: "... except batch dimension".

Reply:
From file `inference-engine/src/legacy_api/src/ngraph_ops/normalize_ie.cpp`:

`NODE_VALIDATION_CHECK(this, input_shape.rank().is_dynamic() || (input_shape.rank().get_length() >= 2 && input_shape.rank().get_length() <= 4), "Argument must have rank >= 2 and <= 4 (argument shape: ", input_shape, ").");`

Looks like now NormalizeIE operation doesn't support 5D tensors. I will update this commentary with this information.

Reply:
Comment was updated.